src/components/ContactContent/ContactContent.jsx

```diff
@@ -62,10 +62,11 @@ const ContactContent = ({numContacts=4}) => {
                 <p className="contactList__desc">
                     All your friends and family financial details in one place. Easily transfer currency internationally at the best possible rates.
                 </p>
+                {!isWallet && 
                 <div className="contactList__search-btn-container">
```

Comment:
contactList__search-btn-container 

this is poor BEM, please revise.

Reply:
What would you like me to change it to?